wasp/src/main/java/com/orhanobut/wasp/IOUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.orhanobut.wasp;
+
+import android.content.Context;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+/**
+ * @author Emmar Kardeslik
+ */
+public class IOUtils {
+
+    private IOUtils() {
+        //no instance
+    }
+
+    public static String readFileFromAssets(final Context context, final String filePath) throws IOException {
+
+        StringBuilder builder = new StringBuilder();
```

Comment:
What do you think if we put a NP control here for context and filepath and if they are null, we can throw an NPE with a convenient message.